app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/subscription/ContactDetailsController.scala

```diff
@@ -119,17 +119,10 @@ class ContactDetailsController @Inject() (
       .map(
         _ =>
           if (inReviewMode) Redirect(DetermineReviewPageController.determineRoute(service))
-          else if (appConfig.contactAddress)
```

Comment:
We can remove appConfig from the file as this is the only place it is used

Reply:
Changes applied, thank you for spotting this out